openapi/active-task-registrations.yaml

```diff
@@ -0,0 +1,43 @@
+get:
+  tags:
+    - "Time registration"
+  operationId: "getTimeRegistrationsByActiveTaskForUser"
```

Comment:
Don't know if i love the name, should we ditch "ForUser" in general for our endpoints? If it's implicitly for the current user, then it's kind of superfluous.

This operation is something about an overview by task, so maybe `getTaskTimeRegistrationsOverview`?